ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/system-control-plane.j2

```diff
@@ -0,0 +1,36 @@
+{% if system.control_plane is arista.avd.defined %}
+
+## System Control-Plane
+{%     if system.control_plane.tcp_mss.ipv4 is arista.avd.defined or system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
+
+#### TCP MSS Ceiling
+
+| Protocol | Segment Size |
+| -------- | -------------|
+{%         if system.control_plane.tcp_mss.ipv4 is arista.avd.defined %}| ipv4 | {{ system.control_plane.tcp_mss.ipv4 }} |
+{%         endif %}
+{%         if system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}| ipv6 | {{ system.control_plane.tcp_mss.ipv6 }} |
+{%         endif %}
+{%     endif %}
+{%     if system.control_plane.ipv4_access_group is arista.avd.defined or system.control_plane.ipv6_access_group is arista.avd.defined %}
+
+#### Control-Plane Access-Groups
+
+| Protocol | VRF | Access-list |
+| -------- | --- | ------------|
+{#         IPv4 Access-groups #}
+{%         for acl_set in system.control_plane.ipv4_access_group | arista.avd.natural_sort %}
+| IPv4 | {%    if acl_set.vrf is arista.avd.defined %}{{ acl_set.vrf }}{% else %}default{% endif %} | {{ acl_set.acl_name }} |
```

Comment:
We try to avoid these inline jinja `if`. I suggest that you replace with `{{ acl_set.vrf | arista.avd.default('default') }}`. Our version of the default filter checks for `None` value as well as `is defined`.
Same for v6

Reply:
Done in 7f64921e53cc2146c20a104a9fb2105c2ad1d79d